faqs/galaxy/datatype_tabular.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Datatype Tabular
+area: Datatypes
+description: Format help for Tabular/BED/Interval Datasets
+layout: faq          
+---
+ 
+
+- A Tabluar datatype is human readable and has tabs seperating data columns.
+  - Note: tabular data is different from comma seperated data (.csv)
+- Common tabular datatypes are .bed, .gtf, .interval, or .txt.
+- The datatype metadata attribute can be directly reassigned to tabular format data.
+- Click the {% icon galaxy-pencil %} icon to reach the **_Edit Attributes_** form. 
```

Comment:
It is best if we also write out the icon description in text, followed by the icon itself, for increased accessibility (e.g. for screen readers)

```suggestion
- Click the pencil icon {% icon galaxy-pencil %} to reach the **_Edit Attributes_** form. 
```